chapter14/dags/00_upload_recipes_to_minio.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from pathlib import Path
+import pendulum
+
+from airflow import DAG
+from airflow.providers.docker.operators.docker import DockerOperator
+from airflow.timetables.trigger import CronTriggerTimetable
+
+with DAG(
+    dag_id="00_upload_recipes_to_minio",
+    schedule=CronTriggerTimetable("@daily", timezone="UTC"),
+    start_date=pendulum.today("UTC"),
+):
+    upload_recipes_to_minio = DockerOperator(
+        task_id="upload_recipes_to_minio",
+        command="upload {{data_interval_start | ds}}",
+        image="vectorvault_cli:latest",
+        docker_url="tcp://docker-socket-proxy:2375",
+        network_mode="chapter14_default",
```

Comment:
We use docker_default in chapter11 and do not provide docker_url there. We mount `/var/run/docker.sock` and that way we do not need the proxy url. Does that work for you too?